packages/tenant-process/src/services/validators.ts

```diff
@@ -55,14 +57,14 @@ export function assertOrganizationVerifierExist(
   }
 }
 
-export function assertExpirationDateExist(
+export function assertExtentionDateExist(
```

Comment:
This function was correct before, this should be `assertExpirationDateExist` and should throw `expirationDateNotFoundInVerifier`.

Why did you change it?

Reply:
there was an overlap of function names between PRs,
The problem is now solved